examples/mobile/HomeApp/index.html

```diff
@@ -178,6 +183,26 @@ <h2>Channel 3</h2>
 					</li>
 				</ul>
 			</div>
+			<div class="ui-popup" id="turn-on-dialog">
+				<div class="ui-popup-header">
+					Add new widget
+				</div>
+				<div class="ui-popup-content">
+					<ul class="ui-listview ui-popup-listview popup-list">
+						<li class="ui-li-has-checkbox">
+								<input type="checkbox" id="latest-news-check"/>
+								<label for="latest-news-ckeck" class="ui-li-text">Latest News</label>
+						</li>
+						<li class="ui-li-has-checkbox">
+								<input type="checkbox" id="now-on-tv-check"/>
+								<label for="now-on-tv-check" class="ui-li-text">Now on TV</label>
+						</li>
+					</ul>
+				</div>
+				<div class="ui-popup-footer">
+					<button class="ui-btn ui-btn-flat" id="popup-submit">OK</button>
```

Comment:
OK is not centered correctly.
Probably you are missing ui-btn-text. Also there should be no lines between items according to the guideline,
See example 
examples/mobile/UIComponents/components/dialogs/dialog.html#?popup=true
![image](https://user-images.githubusercontent.com/2246331/86881256-42948300-c0ee-11ea-8de5-7d5720340001.png)
vs
![image](https://user-images.githubusercontent.com/2246331/86881172-2690e180-c0ee-11ea-8cad-26431cdd57b9.png)

Reply:
Done!